plans/google-analytics-integration.md

```
@@ -0,0 +1,834 @@
+
+CREATE INDEX idx_domain_ga4_configs_property
+ON domain_ga4_configs(ga4_property_id);
+```
+
+---
+
+## Go Implementation
+
+### Package Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)